src/th/nekopost/src/eu/kanade/tachiyomi/extension/th/nekopost/model/SearchRequest.kt

```diff
@@ -2,8 +2,15 @@ package eu.kanade.tachiyomi.extension.th.nekopost.model
 
 import kotlinx.serialization.Serializable
 
+@Serializable
+data class PagingRequest(
```

Comment:
Consider naming this something else. Currently it sounds like a standalone request, not a subset of another request.